packages/apps/reputation-oracle/server/src/modules/abuse/abuse.service.ts

```diff
@@ -168,12 +176,20 @@ export class AbuseService {
           abuseEntity.chainId,
           abuseEntity.escrowAddress,
         );
-        const webhookUrl = (
-          await OperatorUtils.getOperator(
-            abuseEntity.chainId,
-            escrow.exchangeOracle as string,
-          )
-        ).webhookUrl as string;
+        if (!escrow) {
+          throw new Error(
+            `Escrow data is missing. Abuse id: ${abuseEntity.id}`,
+          );
```

Comment:
```suggestion
          throw new Error('Escrow data is missing');
```
and simply log necessary info in `catch` block below